File: src/main/java/no/nav/personopplysninger/features/endreopplysninger/domain/Status.java
package no.nav.personopplysninger.features.endreopplysninger.domain;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import java.util.List;

@JsonIgnoreProperties(ignoreUnknown = true)
public class Status {
    private Integer endringId;
    private String statusType;
    private List<Substatus> substatus;

    public Integer getEndringId() {
        return endringId;
    }

    public String getStatusType() {
        return statusType;
    }

    public List<Substatus> getSubstatus() {
        return substatus;
    }
}

File: src/main/kotlin/no/nav/personopplysninger/features/endreopplysninger/domain/Status.kt
package no.nav.personopplysninger.features.endreopplysninger.domain

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class Status {
    val endringId: Int? = null
    val statusType: String? = null
    var substatus: List<Substatus> = emptyList()
}
